docker/dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:18.04
+
+COPY *.deb /
+
+RUN apt update && \
+  apt install -y curl wget && \
```

Comment:
The EOSIO repos use three-space indentation for C++ source files (dunno why), and four-space indentation for everything else (except YAMLs, I guess). Please change the indentation from two to four spaces to match repo convention. While this may not be as big a deal in our internal repos, the others do care here.

Reply:
Done!